platform/darwin/src/MGLImageSource.h

```diff
@@ -0,0 +1,59 @@
+#import "MGLSource.h"
+
+#import "MGLFoundation.h"
+#import "MGLTypes.h"
+#import "MGLGeometry.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+MGL_EXPORT
+@interface MGLImageSource : MGLSource
+
+#pragma mark Initializing a Source
+
+/**
+ Returns an image source with an identifier and coordinates
```

Comment:
Is it valid to create an image source without a URL or image, or to use it with a raster layer? What does it look like?